common/lib/xmodule/xmodule/mako_module.py

```diff
@@ -39,6 +40,14 @@ def get_context(self):
             'editable_metadata_fields': self.editable_metadata_fields
         }
 
+    def studio_view(self, context):
+        fragment = Fragment(
+            self.system.render_template(self.mako_template, self.get_context())
```

Comment:
This class no longer has any attributes since it's derived from `object`. So this class isn't intended to be instantiated? If so, shouldn't it be made abstract?